ShtefanIlya/HW5/hw_5.2.py

```diff
@@ -0,0 +1,13 @@
+# 0 1 1 2 3 5 8 13
+border = int(input("Enter a border:"))
```

Comment:
look at this code
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])